src/installer/managed/Microsoft.NET.HostModel/Bundle/Bundler.cs

```diff
@@ -38,19 +38,21 @@ public class Bundler
                        BundleOptions options = BundleOptions.None,
                        OSPlatform? targetOS = null,
                        Version targetFrameworkVersion = null,
-                       bool diagnosticOutput = false)
+                       bool diagnosticOutput = false,
+                       string appName = null)
```

Comment:
Why are we adding a parameter if it's not used anywhere?

Reply:
This parameter will be used from the SDK , which will pass the AssemblyName.
I'll change AppName to AssemblyName, so that we don't have to deal with removing `.dll`.

This parameter should be made a required parameter -- but it needs to stay optional until the SDK is updated, so that we don't break SDK builds.